internal/descriptor/table.go

```diff
@@ -109,10 +101,10 @@ func (t *Table[Key, Item]) InsertAt(item Item, key Key) bool {
 	if key < 0 {
 		return false
 	}
-	if diff := int(key) - t.Len(); diff > 0 {
```

Comment:
For example, with 3 items in the table calling `InsertAt(item, 64)` would call `grow(61)` growing the internal slices of masks and items to 61 and 3904. Should be 2 and 128.